yt_dlp/downloader/websocket.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import division, unicode_literals
+
+import os
+import signal
+import asyncio
+import threading
+try:
+    import websockets
+    has_websockets = True
+except ImportError:
+    has_websockets = False
+
+from .common import FileDownloader
+from .external import FFmpegFD
+
+
+class FFmpegSinkFD(FileDownloader):
+    """ A sink to ffmpeg for downloading fragments in any form """
+
+    def real_download(self, filename, info_dict):
+        info_dict = info_dict.copy()
+        info_dict['url'] = '-'
```

Comment:
oh, don't do that! It's completely removing WebSocket URL!

Reply:
sorry, fixed

Reply:
thanks